templates/edge-computing/index.html

```diff
@@ -391,28 +458,6 @@ <h2>What is the size of a micro cloud?</h2>
   </div>
 </section>
 
-<section class="p-strip--suru-accent is-dark">
-  <div class="u-fixed-width">
-    <div class="p-heading-icon--small">
-      <div class="p-heading-icon__header">
-        {{ image (
-          url="https://assets.ubuntu.com/v1/4ab8ff35-Whitepaper+-+white.svg",
-          alt="",
-          width="171",
-          height="150",
-          hi_def=True,
-          loading="lazy"
-          ) | safe
-        }}
-        <p class="p-heading--4">Whitepaper</p>
-      </div>
-      <h2 class="p-heading--4">CTOs' Guide to Micro Clouds</h2>
-      <p>This whitepaper demonstrates to Technical Leads the concept of micro clouds, a new class of compute for the edge and decentralised environments. Access the report today to get a condensed work of understanding edge computing constraints, values, graphics, and use cases.</p>
-      <p><a href="/engage/edge-infrastructure" class="p-button">Download the whitepaper</a></p>
-    </div>
-  </div>
-</section>
-
 <section class="p-strip u-sv3">
```

Comment:
We should probably add a divider here, the CTA section is supposed to be separate from the one above if I'm understanding this layout correctly